charts/sn-platform/templates/proxy/proxy-cluster.yaml

```diff
@@ -57,7 +57,7 @@ spec:
 {{- with .Values.proxy.labels }}
 {{ toYaml . | indent 6 }}
 {{- end }}
-{{- if or (and .Values.prometheus.scrape.proxy .Values.prometheus.scrape.enabled) .Values.monitoring.datadog .Values.proxy.annotations }}
+{{- if or (and .Values.prometheus.scrape.proxy .Values.prometheus.scrape.enabled) .Values.monitoring.datadog .Values.proxy.annotations .Values.auth.oauth.enabled .Values.auth.authentication.jwt.enabled }}
```

Comment:
It seems the `Values.auth.oauth.enabled` and `.Values.auth.authentication.jwt.enabled` are put to the wrong place

Reply:
It should be at line 73?

Reply:
Great catch, thanks, PTAL @FushuWang